_drafts/oryx/versioning.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Versioning
+description: Versioning
+template: concept-topic-template
+---
+
+# Versioning
+In this document, we outline the methods we use to deliver an advanced application development platform while maintaining stability. Our goal is to ensure that any upcoming changes are introduced in a predictable manner. We want all Oryx users to be informed about the addition of new features and adequately prepared for the removal of outdated ones.
+
+Stability is a critical aspect of any software framework, and it's especially important for frontend frameworks like Oryx. The stability of a framework refers to its ability to function reliably and consistently over time, despite changes in the environment or updates to its components. 
+
+Ensuring that a framework is stable requires careful planning, testing, and versioning. In this article, we will explore how the Oryx framework achieves stability through its versioning and release processes. We'll look at how Oryx manages its releases, how it uses version numbers to indicate changes, and how it ensures backward compatibility for its users. 
+
```

Comment:
Wondering, if you should not mention about rc versions. As it's related to both mentioned releasing and stability.

Reply:
Good point.